build.gradle.kts

```diff
@@ -150,9 +150,6 @@ subprojects {
   configure<SpotlessExtension> {
     kotlin {
       target("**/*.kt")
-      ktlint(libs.versions.ktlint.get()).userData(mapOf("indent_size" to "2"))
```

Comment:
You still need `ktlint()` here

Reply:
Very nice catch. I might be soured on DSLs.

Reply:
Only know because I did it first!